src/ashell/shell-state.c

```diff
@@ -282,37 +272,38 @@ int32_t ashell_print_file(char *buf)
 
     /* Error getting an id for our data storage */
     if (!file) {
-        comms_println(ERROR_FILE_NOT_FOUND);
+        comms_print(ERROR_FILE_NOT_FOUND);
         return RET_ERROR;
     }
 
     ssize_t size = fs_size(file);
     if (size == 0) {
-        comms_println("Empty file");
+        comms_print("Empty file\r\n");
         fs_close_alloc(file);
         return RET_OK;
     }
 
     fs_seek(file, 0, SEEK_SET);
-    if (lines)
-        comms_printf("%5d  ", line++);
 
+    int lineStart = 0;
     do {
-        count = fs_read(file, data, 4);
```

Comment:
Rather than reading and printing one character at a time, it does up to 80.  Note comms_printf is no longer used for critical output as it will occasionally add extra characters.